glsl/Gadgets.frag

```diff
@@ -0,0 +1,61 @@
+#include "IECoreGL/FilterAlgo.h"
+#include "IECoreGL/ColorAlgo.h"
+
+uniform bool border;
+uniform vec2 borderRadius;
+uniform float borderWidth;
+
+uniform bool edgeAntiAliasing;
+
+uniform int textureType;
+uniform sampler2D texture;
+
+#if __VERSION__ >= 330
+
+uniform uint ieCoreGLNameIn;
+layout( location=0 ) out vec4 outColor;
+layout( location=1 ) out uint ieCoreGLNameOut;
+#define OUTCOLOR outColor
+
+#else
+
+#define OUTCOLOR gl_FragColor
+
+#endif
+
+void main()
```

Comment:
This shader now appears in the Scene/OpenGL/Shaders submenu! I don't think we want that. Also, I think StandardStyle.frag might be a more descriptive name...